src/components/Layout/Footer.jsx

```diff
@@ -32,53 +36,57 @@ function FooterContent() {
 function SubFooter() {
   const { Footer } = Layout;
   return (
-    <Footer style={{ backgroundColor: '#403737' }}>
+    <Footer
+      style={{
+        backgroundColor: primary_accent_color,
+      }}
+    >
       <Space direction="horizontal">
         <Button
```

Comment:
Would it be possible to map through an object for this data? Might reduce the lines of codes significantly.

Reply:
Sorry, I'm having trouble picturing what you're imagining; could you elaborate a little?

Reply:
Oh, never mind - now I see the repeated UI you are pointing at! Yes, we can for sure map to generate the footer components.

Reply:
It is done!

Reply:
Thank you Mack!